### DISCUSSION TITLE
Can we have an option to disable the dots?

### DISCUSSION THREAD

**Discussion by DaveHenning** [liamcain/obsidian-calendar-plugin#72] [Ideas]
When using Daily Notes, the amount I write isn't really relevant to anything. I don't write long form on daily notes, It's typically just single sentences or links to other notes. I think an option to disable the dots would keep the calendar looking clean for people like me. 

Selfish request: If I could track h4's, instead of wordcount, that would be helpful. I'm sure I'm in the minority on the h4 usage, so maybe a more general all headings count, or maybe a property selector (count h4 or h2, etc.) Just a thought.

Love the plugin. It's easily my most used. Thank you.

**Comment by liamcain** [comment]
You can [disable the dots entirely](https://github.com/liamcain/obsidian-calendar-plugin#settings) by setting "Words Per Dot" to zero. 

The option to track headings instead is interesting though, I haven't heard that request yet! I have seen people changing the Words Per Dot to a very low value (e.g. 10 words per dot) to give a rough approximation for the number of links in their daily notes. That might be worth a try for your case.

There is definitely room for more configuration with the display so I'm always interested in hearing how people interact with the calendar and what their workflows is!

**Reply by DaveHenning** [reply]
> You can disable the dots entirely by setting "Words Per Dot" to zero.

Ah, you're right. When I tried that I still saw the empty dots (for checkboxes) and just assumed it didn't work. This was before I knew what the empty circles meant. I have set it back to zero. Still would love the option to disable so the calendar can be even cleaner (I only sometimes have checkboxes on my dailies, and don't really care to track those). I tried a low number for a bit and it still didn't have much value for me. Again, I assume my use case is not likely the norm. 

As far as tracking headings go, a brief example of how my daily works:


# 2020.12.12 Saturday
[[_dailies/2020.12.11|<< Yesterday]] | [[_dailies/2020.12.13|Tomorrow >>]]
----
## Log
#### 2:30 PM - Explaining Daily Notes
Lorem Ipsum. 
#### 2:35 PM - This happened
Lorem Ipsum Dolor.

----

I skip the h3 for two reasons a) to allow for a (rarely used) level between the Log and the timestamps and b) h3 was too large. 

I like using headings (instead of just bolding, which I toyed with) because it's much easier to link back to it later.

**Reply by liamcain** [reply]
If you want to hide the tasks as well as the dots, you can add the following as a CSS snippet:

```css
#calendar-container .dot-container {
  display: none;
}
```

You can save that as `calendar.css` in your `snippets/` folder - 
<img width="1138" alt="css snippets" src="https://user-images.githubusercontent.com/693981/101995855-25ff0700-3c9b-11eb-8806-07c019e1f32c.png">

Note: this will hide _all_ dots and make the calendar a bit smaller.

**Reply by liamcain** [reply]
Also, I appreciate the insight into your daily notes workflow!

**Reply by DaveHenning** [reply]
I figured there was some CSS to hide them, but wanted to see about the toggle first. Thanks for pulling that together for me. So clean now. 

If you ever add a 'count headings instead of text' option, I'll surely be a guinea pig if you need one.
